crates/biome_js_analyze/tests/specs/complexity/useLiteralKeys/invalid.js.snap

```diff
@@ -600,11 +615,109 @@ invalid.js:34:25 lint/complexity/useLiteralKeys  FIXABLE  ━━━━━━━
     33 │ // optional chain
   > 34 │ a?.["b"]?.['c']?.d?.e?.["f"]
        │                         ^^^
-    35 │ 
+    35 │ a = {
+    36 │   ["line1\
   
   i Unsafe fix: Use a literal key instead.
   
     34 │ a?.["b"]?.['c']?.d?.e?.["f"]
        │                        -- --
 
 ```
+
+```
+invalid.js:36:4 lint/complexity/useLiteralKeys  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! The computed expression can be simplified without the use of a string literal.
```

Comment:
The message here doesn't fit well into object keys. Do you think we should change it to: 
```
The computed expression can be simplified to a string literal.
```
@Conaclos

Reply:
This could be a nice improvement!